Dockerfile

```diff
@@ -27,4 +27,4 @@ USER $USERNAME
 
 RUN [ "yarn", "run", "build-css" ]
 
-CMD [ "yarn", "start" ]
+CMD [ "node", "app.js" ]
```

Comment:
Can a comment be added here to explain why this is as it is so nobody tries to change it back to using Yarn to run something? Thanks!